packages/loader/container-loader/src/protocol.ts

```diff
@@ -30,6 +30,13 @@ export enum SignalType {
 	Clear = "clear", // used only by client for synthetic signals
 }
 
+export interface ILocalSequencedClient extends ISequencedClient {
```

Comment:
We should try and document interfaces in the process and be clear on the intention. It may help us reason later if we need this interface at all or ISequencedClient can be extended with optional param.

Reply:
Looking at Matt's comments, this interface should go away with the work item he added. Maybe worth noting it in the description and adding work item AB.